pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -517,24 +516,24 @@ public void addUnAckedMessages(int numberOfMessages) {
     }
 
     public boolean isBlockedDispatcherOnUnackedMsgs() {
-        return BLOCKED_DISPATCHER_ON_UNACKMSG_UPDATER.get(this) == TRUE;
+        return blockedDispatcherOnUnackedMsgs == TRUE;
     }
-    
+
     public void blockDispatcherOnUnackedMsgs() {
-        BLOCKED_DISPATCHER_ON_UNACKMSG_UPDATER.set(this, TRUE);
+        blockedDispatcherOnUnackedMsgs = TRUE;
```

Comment:
why are we not using updater for setting the value?

Reply:
Same as for get, updated is not needed. We only need updated (or Atomic) for compareAndSet or increment operations

Reply:
typo: we only need the *updater* for compareAndSet.. 

@rdhabalia this changes have no functional impact. if there's a concern I can remove, but I think that's more readable without the updater when possible.